lib/animina/accounts/resources/fast_user.ex

```diff
@@ -0,0 +1,116 @@
+defmodule Animina.Accounts.FastUser do
+  @moduledoc """
+  This is the Fast User module.
+  """
+
+  use Ash.Resource,
+    data_layer: AshPostgres.DataLayer,
+    authorizers: Ash.Policy.Authorizer,
+    domain: Animina.Accounts,
+    extensions: []
+
+  postgres do
+    table "users"
+    repo Animina.Repo
+  end
+
+  actions do
+    defaults []
+
+    action :by_username, {:array, :map} do
+      argument :username, :string, allow_nil?: false, public?: true
+
+      run fn input, _ ->
+        {:ok, []}
+      end
+    end
+  end
+
+  attributes do
+    uuid_primary_key :id
+    attribute :email, :ci_string, allow_nil?: false, public?: true
+    attribute :hashed_password, :string, allow_nil?: false, sensitive?: true
```

Comment:
Let's keep this to the minimum which we need for the profile view. Which means e.g. no need for the `:hashed_password` and other attributes. Strip it down as fare as possible.